### PR TITLE
Add invalidation

### PR DESCRIPTION
**Pull request by TorbenVallund**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,6 +289,24 @@ A callback will be made on the AddressCallbackUrl when the user swipes to accept
...
+
+The invalidation request will be processed in the MobilePay backend according to these rules:
+1. If a successful authorization already exists on the payment, the invalidation will return an error.
+2. If a callback has been sent, we will wait 30 seconds for a patch to the authorization attempt. A failed authorization will result in an invalidation, but a successful authorization will return an error.
+3. If we do not receive the patch within 30 seconds, we will accept the invalidation.
+4. When the invalidation is completed, the user cannot request or accept this payment.
+
+### Invalidation before callback
+If the invalidation request is sent before the callback has been received, the callback will not be sent and all that is needed is to call the invalidation endpoint.
```
Comment (BenjaminGraugaard) [inline README.md:303]: We need to be more precise here. Suggestion:
If the invalidation request is received by MobilePay before the callback to the PSP is made, no callback will be performed and the invalidation will complete almost immediately.

File: README.md [hunk 2]
```diff
@@ -289,6 +289,24 @@ A callback will be made on the AddressCallbackUrl when the user swipes to accept
...
+![Invalidation before callback](./assets/invalidation-before-callback.svg)
+
+### Invalidation after callback
+If the invalidation request is sent after the callback has been received, it is needed to call the invalidation endpoint as well as to patch the authorization attempt with error code 1010 - RejectedForInvalidatePayment.
```
Comment (BenjaminGraugaard) [inline README.md:307]: to be more clear:
If the invalidation request is received by MobilePay after the callback to the PSP is performed, the PSP must also PATCH the authorizationAttempt with succeeded=false and the new ReasonCode 1010 - RejectedForInvalidatePayment.

File: README.md (hunk 1 above)
Comment (TorbenVallund) [reply README.md]: Fixed in commit 2

File: README.md (hunk 2 above)
Comment (TorbenVallund) [reply README.md]: Fixed in commit 2. I removed "new" from your suggestion.

**Pull request merged**